tests/test_vsa.py

```diff
@@ -193,480 +170,579 @@ def test_wrapped_intervals(self):
+        self.si_e = self.SI(bits=16, stride=1, lower_bound=0x2000, upper_bound=0x3000)
+        self.si_f = self.SI(bits=16, stride=1, lower_bound=0, upper_bound=255)
+        self.si_g = self.SI(bits=16, stride=1, lower_bound=0, upper_bound=0xFF)
+        self.si_h = self.SI(bits=32, stride=0, lower_bound=0x80000000, upper_bound=0x80000000)
+
+    def is_equal(self, ast_0, ast_1):
+        return claripy.backends.vsa.identical(ast_0, ast_1)
+
+    # Check this
+    def test_foo(self):
         si1 = claripy.TSI(32, name="foo", explicit_name=True)
         assert vsa_model(si1).name == "foo"
 
-        # Normalization
-        si1 = SI(bits=32, stride=1, lower_bound=10, upper_bound=10)
-        assert vsa_model(si1).stride == 0
-
-        # Integers
-        si1 = claripy.SI(bits=32, stride=0, lower_bound=10, upper_bound=10)
-        si2 = claripy.SI(bits=32, stride=0, lower_bound=10, upper_bound=10)
-        si3 = claripy.SI(bits=32, stride=0, lower_bound=28, upper_bound=28)
-        # Strided intervals
-        si_a = claripy.SI(bits=32, stride=2, lower_bound=10, upper_bound=20)
-        si_b = claripy.SI(bits=32, stride=2, lower_bound=-100, upper_bound=200)
-        si_c = claripy.SI(bits=32, stride=3, lower_bound=-100, upper_bound=200)
-        si_d = claripy.SI(bits=32, stride=2, lower_bound=50, upper_bound=60)
-        si_e = claripy.SI(bits=16, stride=1, lower_bound=0x2000, upper_bound=0x3000)
-        si_f = claripy.SI(bits=16, stride=1, lower_bound=0, upper_bound=255)
-        si_g = claripy.SI(bits=16, stride=1, lower_bound=0, upper_bound=0xFF)
-        si_h = claripy.SI(bits=32, stride=0, lower_bound=0x80000000, upper_bound=0x80000000)
-
-        assert is_equal(si1, claripy.SI(bits=32, to_conv=10))
-        assert is_equal(si2, claripy.SI(bits=32, to_conv=10))
-        assert is_equal(si1, si2)
-        # __add__
-        si_add_1 = si1 + si2
-        assert is_equal(si_add_1, claripy.SI(bits=32, stride=0, lower_bound=20, upper_bound=20))
-        si_add_2 = si1 + si_a
-        assert is_equal(si_add_2, claripy.SI(bits=32, stride=2, lower_bound=20, upper_bound=30))
-        si_add_3 = si_a + si_b
-        assert is_equal(si_add_3, claripy.SI(bits=32, stride=2, lower_bound=-90, upper_bound=220))
-        si_add_4 = si_b + si_c
-        assert is_equal(si_add_4, claripy.SI(bits=32, stride=1, lower_bound=-200, upper_bound=400))
+    def test_normalization(self):
+        # Test for Normalization
+        si1_normalized = self.SI(bits=32, stride=1, lower_bound=10, upper_bound=10)
+        assert vsa_model(si1_normalized).stride == 0
+
+    def test_integers(self):
+        # Test for Integers
+        assert self.is_equal(self.si1, self.SI(bits=32, to_conv=10))
+        assert self.is_equal(self.si2, self.SI(bits=32, to_conv=10))
+        assert self.is_equal(self.si1, self.si2)
+
+    def test_strided_intervals(self):
+        # Strided interval assertions could go here if needed in future
```

Comment:
I appreciate the forward thinking but if the test is empty let's remove it for now, we can always add it back